<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(38, 38, 38);
}</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="switch_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: none;
	background-color: rgb(38, 38, 38);
	color : rgb(226, 194, 255);
	font: 34pt &quot;Anta&quot;;
	border-radius: 1px;
}
QPushButton:hover{
	border: 2px solid rgb(226, 194, 255);
	background-color: rgb(38, 38, 38);
	border-radius: 1px;
}
QPushButton:pressed{
	border: 2px solid rgb(242, 247, 242);
	color: rgb(242, 247, 242);
	border-radius: 1px;
}
</string>
    </property>
    <property name="text">
     <string>Atbash</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="alphabet_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>480</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiLight</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: rgb(255, 170, 255)
}</string>
    </property>
    <property name="text">
     <string>абвгдеёжзийклмнопрстуфхцчшщъыьэюя</string>
    </property>
   </widget>
   <widget class="QLabel" name="alphabet_label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>110</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Алфавит</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="plain_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>480</width>
      <height>170</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
	border: 2px solid rgb(71, 61, 80);
	border-radius: 1px;
	color: rgb(201, 192, 255);
	selection-background-color: rgb(56, 56, 56);
}
QTextEdit:hover{
	border: 2px solid rgb(226, 194, 255);
}
QTextEdit:focus{
	border: 2px solid rgb(255, 255, 255);	
	background-color: rgb(43, 43, 43);
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="placeholderText">
     <string>Plain text</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="crypted_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>480</width>
      <height>170</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
	border: 2px solid rgb(97, 117, 84);
	border-radius: 1px;
	color: rgb(208, 239, 177);
	selection-background-color: rgb(56, 56, 56);
}
QTextEdit:hover{
	border: 2px solid rgb(226, 194, 255);
}
QTextEdit:focus{
	border: 2px solid rgb(255, 255, 255);	
	background-color: rgb(43, 43, 43);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="placeholderText">
     <string>Crypted text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switch_lg_btn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>157</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Сменить</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
